src/core/mapping/machine.cc

```diff
@@ -263,7 +263,7 @@ LocalProcessorRange Machine::slice(TaskTarget target,
       return LocalProcessorRange();
   }
 
-  return LocalProcessorRange(slice.low - global_range.low,
+  return LocalProcessorRange(slice.low - global_range.low + machine_desc.processor_range().low,
```

Comment:
I believe `slice.low - global_range.low + machine_desc.processor_range().low` is the same as `slice.low`.

Reply:
yes, thank you.